runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -577,5 +608,15 @@ private void resetTimers()
 		{
 			lastInteract = null;
 		}
+
+		// Reset fishing idle timer
```

Comment:
You could've avoided this if you just unified combat idle with your new functionality